Cache/CacheSubscriber.php

```diff
@@ -64,9 +66,10 @@ public function getEvents() {
 
     /**
      * @param HasEmitterInterface $client
-     * @param null $options
+     * @param null                $options
      */
-    public function attach(HasEmitterInterface $client, $options = null) {
+    public function attach(HasEmitterInterface $client, $options = null)
+    {
         if (!isset($options['storage'])) {
             $options['storage'] = new CacheStorage(new ArrayCache());
```

Comment:
Il problema è qui da quello che ho capito.
Utilizzando l'`ArrayCache` la pagina viene memorizzata in un array, quindi alla seconda richiesta non è presente.
Inoltre non mi è chiaro se e come si possa ridefinire `$options['storage']` nel codice dell'utente.